packages/core/src/odata-common/request-builder/request-builder-base.ts

```diff
@@ -132,6 +132,18 @@ export abstract class MethodRequestBuilder<
     return this;
   }
 
+  /**
+   * Append the given path to the URL.
```

Comment:
Explain in the docs that all strings are simply joined and appended to the path. No logic like adding slashes etc. is performend.